src/content/developers/docs/consensus-mechanisms/pos/index.md

```diff
@@ -3,85 +3,64 @@ title: Proof-of-stake (PoS)
-| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- |
-| Staking makes it easier for you to run a node. It doesn't require huge investments in hardware or energy, and if you don't have enough ETH to stake, you can join staking pools.                                                                                          | Proof-of-stake is still in its infancy, and less battle-tested, compared to proof-of-work |
-| Staking is more decentralized. It allows for increased participation, and more nodes doesn't mean increased % returns, like with mining.                                                                                                                                  |                                                                                           |
-| Staking allows for secure sharding. Shard chains allow Ethereum to create multiple blocks at the same time, increasing transaction throughput. Sharding the network in a proof-of-work system would simply lower the power needed to compromise a portion of the network. |                                                                                           |
+| Pros                                                                                                                                                                                                                | Cons                                                                                                 |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
+| Staking makes it easier for individuals to participate in securing the network, promoting decentralization. validator node can be run on a normal laptop. Staking pools allow users to stake without having 32 ETH. | Proof-of-stake is younger and less battle-tested compared to proof-of-work                                               |
+| Staking is more decentralized. It allows for increased participation, and more nodes doesn't mean increased % returns, whereas more hashpower means higher % returns with proof-of-work mining.                               | Proof-of-stake is more complex to implement than proof-of-work                                                            |
```

Comment:
Would argue more hash power = more returns just like more staked ETH = more returns... The thing to note here is PoS doesn't really enable economies of scale, as ASICs are expensive, and you can get significant discounts when buying bulk... and opportunity the little guy cannot take advantage of. PoS has reasonably low fixed costs (1 computer) but after that, the rate of return on your ETH is basically the same.

Reply:
fair point. updated accordingly.